HospitalFrontend/src/views/ObserveFeedback.vue

```diff
@@ -1,10 +1,45 @@
 <template>
-  <h1>This is a ObserveFeedback page for any user and admin </h1>
+  <h1>User Feedbacks</h1>
+  <div id="feedbackList" class="list-group container w-50" >
+       <FeedbackItem v-for="feedback in feedbacks" v-bind:key="feedback.Id" v-bind:feedback="feedback"></FeedbackItem>
+  </div>
 </template>
 
 <script>
+
+import FeedbackItem from '../components/FeedbackItem.vue'
+import axios from 'axios';
+
+
 export default {
-  name:'ObserveFeedback'
+  name:'ObserveFeedback',
+  data: function()
+  {
+      return{
+        feedbacks:[]
+      }
+  },
+  components:
+  {
+     FeedbackItem
+  },
+  methods: 
+  {
+
+    getAllFeedbacks:function()
+    {
+      axios.get("http://localhost:5290/feedback").then((response) => 
```

Comment:
Defining a direct path can cause an error to other developers @dracdev

Reply:
Fixed in collaboration with other developers